cmd/swarmd/main.go

```diff
@@ -238,6 +251,7 @@ func init() {
 	mainCmd.Flags().String("listen-debug", "", "Bind the Go debug server on the provided address")
 	mainCmd.Flags().String("listen-metrics", "", "Listen address for metrics")
 	mainCmd.Flags().String("join-addr", "", "Join cluster with a node at this address")
+	mainCmd.Flags().String("generic-node-resource", "", "user defined resources (e.g. fpga=2;gpu={UUID1,UUID2,UUID3})")
```

Comment:
My 2 cents: In docker we have CPU sets and the syntax is `--cpuset-cpus="1,3"`.

Maybe we should use something similar here?